lifelines/statistics.py

```diff
@@ -157,6 +132,7 @@ def multivariate_logrank_test(event_durations, groups, event_observed=None,
       test_result: True if reject the null, (pendantically) None if we can't reject the null.
 
     """
+    event_durations, groups = np.asarray(event_durations), np.asarray(groups)
```

Comment:
`multivariate_logrank_test` would fail it I put into python lists, so this changes that.